pup-0001.md

```diff
@@ -0,0 +1,269 @@
+---
+title: PUP Purpose and Guidelines
+author: Brian Bouterse
+created: 13-Feb-2017
+status: Approved
+---
+
+Summary
+=======
+
+This proposal provides a structured process to propose, track, discuss, and decide on process
+changes for the Pulp project and community.
+
+
+Motivation
+==========
+
+As initially [raised on the pulp-dev list](https://www.redhat.com/archives/pulp-dev/2017-February/msg00014.html),
+Pulp lacks a process to propose, track, discuss, and decide on changes that affect the Pulp
+community.
+
+This is a problem that has been addressed by many communities. The process outlined in this document
+is inspired by processes in the Python, Django, and Foreman communities.
+
+
+What are these called?
+======================
+
+These documents are called Pulp Update Proposals or a "PUP" for short.
+
+
+Tracking PUPs
+=============
+
+An index mimicking [the Python PEP index](https://www.python.org/dev/peps/) tracks all approved,
+rejected, and abandoned PUPs.
+
+
+PUP Process
+===========
+
+The overall process is: Pre-proposal -> Authoring -> Announcement ->  Discussion and Feedback ->
+Revision -> Voting -> Decision
+
+Most of the process is facilitated using either the pulp-list or pulp-dev mailing lists. The
+following guidelines identify how they should.
+
+* For changes impacting users, [pulp-list](https://www.redhat.com/mailman/listinfo/pulp-list) is the
+  appropriate place.
+* For changes impacting developers, [pulp-dev](https://www.redhat.com/mailman/listinfo/pulp-dev) is
+  the appropriate place.
+
+
+Pre-Proposal
+^^^^^^^^^^^^
```

Comment:
Fixed in next push and I'm proofreading the page myself too using the github markdown this time. Thank you.